Khan/wonder-blocks
# Update Link WithTypography story

## Summary:
Updated Link's WithTypography story and description to recommend always using `Typography` as the parent element of `Link`. This is a temporary fix for the styling issues caused when `Link` wraps a `Typography` element instead.

Issue: WB-1548

## Test plan:
- In Storybook, go to Link's [With Typography story](https://5e1bf4b385e3fb0020b7073c-vscacbzcbc.chromatic.com/?path=/docs/link--with-typography).
- Check Docs for description.
- Click "Show code" to see implementation.

  <img width="628" alt="Screenshot 2023-05-10 at 2 25 30 PM" src="https://github.com/Khan/wonder-blocks/assets/100858764/12b35a2d-cfaa-4695-8594-87c9af8f561e">
